crosschain/simple/src/main/java/tech/pegasys/samples/crosschain/simple/transaction/CrosschainTransactionNoParams.java

```diff
@@ -1,4 +1,331 @@
+                this.web3jSc0, SC0_SIDECHAIN_ID, this.coordinationContractSetup.getCrosschainCoordinationContractAddress(), CROSSCHAIN_TRANSACTION_TIMEOUT);
+        this.tmSc2 = new CrosschainTransactionManager(this.web3jSc2, this.credentials, SC2_SIDECHAIN_ID, RETRY, POLLING_INTERVAL,
+                this.web3jSc0, SC0_SIDECHAIN_ID, this.coordinationContractSetup.getCrosschainCoordinationContractAddress(), CROSSCHAIN_TRANSACTION_TIMEOUT);
+
+        // Hyperledger Besu is configured as an IBFT2, free gas network. We need a free gas provider.
+        this.freeGasProvider = new StaticGasProvider(BigInteger.ZERO, DefaultGasProvider.GAS_LIMIT);
+    }
+
+    private void deployAndSetupCoordinationContract() throws Exception {
+        this.coordinationContractSetup.deployCrosschainCoordinationContract();
+    }
+
+    private void loadContracts() {
+        LOG.info("Loading contracts");
+        LOG.info(" Contract 1: {}", this.contract1Address);
+        LOG.info(" Contract 2: {}", this.contract2Address);
+
+        this.contract1 = Sc1Contract1.load(this.contract1Address, this.web3jSc1, this.tmSc1, this.freeGasProvider);
+        this.contract2 = Sc2Contract2.load(this.contract2Address, this.web3jSc2, this.tmSc2, this.freeGasProvider);
+    }
+
+    private void deployContracts() throws Exception {
+        LOG.info("Deploying contracts");
+        RemoteCall<Sc2Contract2> remoteCallContract2 =
+                Sc2Contract2.deployLockable(this.web3jSc2, this.tmSc2, this.freeGasProvider);
+        this.contract2 = remoteCallContract2.send();
+        this.contract2Address = this.contract2.getContractAddress();
+        LOG.info(" Contract 2 deployed on sidechain 2 (id={}), at address: {}",  SC2_SIDECHAIN_ID, contract2Address);
+
+        RemoteCall<Sc1Contract1> remoteCallContract1 =
+                Sc1Contract1.deployLockable(this.web3jSc1, this.tmSc1, this.freeGasProvider, SC2_SIDECHAIN_ID, contract2Address);
+        this.contract1 = remoteCallContract1.send();
+        this.contract1Address = this.contract1.getContractAddress();
+        LOG.info(" Contract 1 deployed on sidechain 1 (id={}), at address: {}", SC1_SIDECHAIN_ID, contract1Address);
+    }
+
+
+
+
+    private void core() throws Exception {
+        LOG.info("Running Core Part of Sample Code");
+
+        LOG.info(" Set state in each contract to known values that aren't zero.");
+        TransactionReceipt transactionReceipt;
+        LOG.info("  Single-chain transaction: Contract2.clear()");
+        transactionReceipt = this.contract2.clear().send();
+        assertTrue(transactionReceipt.isStatusOK());
+
+        checkExpectedValues(false);
+
+        Scanner myInput = new Scanner( System.in );
+        Boolean c2Val = this.contract2.get().send();
+
+        LOG.info("  Executing call simulator to determine parameter values and expected results");
+        CallSimulator sim = new CallSimulator();
```

Comment:
given there are no parameters passed, and no values returned, there is no need for a simulator